src/SFA.DAS.Admin.Aan.Web/Controllers/ManageEvent/NetworkEventOrganiserNameController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using FluentValidation;
+using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Admin.Aan.Application.Services;
+using SFA.DAS.Admin.Aan.Web.Authentication;
+using SFA.DAS.Admin.Aan.Web.Infrastructure;
+using SFA.DAS.Admin.Aan.Web.Models.NetworkEvent;
+
+namespace SFA.DAS.Admin.Aan.Web.Controllers.ManageEvent;
+
+[Authorize(Roles = Roles.ManageEventsRole)]
+[Route("events/new/organiser", Name = RouteNames.ManageEvent.EventOrganiserName)]
+public class NetworkEventOrganiserNameController : Controller
+{
+
+    private readonly ISessionService _sessionService;
+    private readonly IValidator<EventOrganiserNameViewModel> _organiserNameValidator;
+
+    public NetworkEventOrganiserNameController(ISessionService sessionService, IValidator<EventOrganiserNameViewModel> organiserNameValidator)
+    {
+        _sessionService = sessionService;
+        _organiserNameValidator = organiserNameValidator;
+    }
+
+    public const string OrganiserNameViewPath = "~/Views/ManageEvent/EventOrganiserName.cshtml";
+
+
+    [HttpGet]
+    public IActionResult Get()
+    {
+        var sessionModel = _sessionService.Get<EventSessionModel>();
+        var augmentedModel = GetOrganiserNameViewModel(sessionModel);
+
+        return View(OrganiserNameViewPath, augmentedModel);
+    }
+
+    [HttpPost]
+    public IActionResult Post(EventOrganiserNameViewModel submitModel)
+    {
+        var result = _organiserNameValidator.Validate(submitModel);
+
+        var sessionModel = _sessionService.Get<EventSessionModel>();
+        sessionModel.OrganiserName = submitModel.OrganiserName;
+        sessionModel.OrganiserEmail = submitModel.OrganiserEmail;
```

Comment:
This should be moved further down as it is only applicable if validation has passed.

Reply:
I see that you are passing the session model for rehydration in case validation fails but you dont need to, see my comments above.

Reply:
I was using the sessionmodel as the hydration method in both the get and set so that it it much more easily reusable when we come to start using the CheckYourAnswers links.  Lets discuss.

Reply:
ok this is done, as discussed